### PR TITLE
azure.rst: Editorial changes

### PR DESCRIPTION
**Pull request by DougTW**
Editorial changes include:
- added standardized introduction paragraph, description, and prerequisites
- rearranged some content into introduction
- revised first usage of Microsoft Azure to have disclaimer asterisk
- added contents directive at the start of the document
Signed-off-by: DougTW <doug.martin@intel.com>

### CODE REVIEW

File: source/get-started/cloud-install/azure.rst [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
-############################
+|CL-ATTR| on Microsoft\* Azure\*
+################################
 
-|CL-ATTR| is available for you to use in the Microsoft* Azure* marketplace and
+The Azure :abbr:`CLI (Command Line Interface)` offers the ability to create and
+manage resources in MS Azure from the command line. In this tutorial you learn
+to:
+
+#. Install the latest MS Azure CLI on your |CL| machine.
```
Comment (mvincerx) [inline source/get-started/cloud-install/azure.rst]: Remove these steps. The contents directive creates a TOC, which presents same info.

File: source/get-started/cloud-install/azure.rst [hunk 2]
```diff
@@ -18,25 +41,12 @@ is offered with three different images, also known as a
...
-#. Install the latest MS Azure CLI on your |CL| machine.
-
-#. Log into MS Azure using the CLI 2.0 interface.
-
-#. Create a MS Azure resource group.
-
-#. Locate the |CL| images.
+`Azure portal`_ or by using the MS Azure CLI 2.0. If you do not already have an
+account set up with MS Azure, you can sign up for a `MS Azure free account`_
+to access the |CL| :abbr:`VM (Virtual Machine)` images.
```
Comment (mvincerx) [inline source/get-started/cloud-install/azure.rst]: Sign in at the `Azure portal`_ to access these images from the Azure dashboard, or use the MS Azure CLI 2.0.

File: source/get-started/cloud-install/azure.rst [hunk 3]
```diff
@@ -286,13 +296,13 @@ Log into your Microsoft Azure account
...
 
-#. Once you enter the authentication code, the website changes to a screen to
+#. Once you enter the authentication code, the website displays a screen to
    enter your existing Microsoft Azure credentials.
 
 #. Log in with your existing MS Azure account credentials. Once complete, the
-   browser screen changes again as shown in figure 3. The screen shows you
-   have signed into the Microsoft Cross-platform Command Line Interface
-   application on your device and you can close the window.
+   browser screen shows you have signed into the Microsoft Cross-platform
```
Comment (mvincerx) [inline source/get-started/cloud-install/azure.rst]: Remove "Once complete."

Comment (DougTW) [reply source/get-started/cloud-install/azure.rst]: Done.

File: source/get-started/cloud-install/azure.rst (hunk 2 above)
Comment (DougTW) [reply source/get-started/cloud-install/azure.rst]: Done.

File: source/get-started/cloud-install/azure.rst (hunk 1 above)
Comment (DougTW) [reply source/get-started/cloud-install/azure.rst]: Done.

File: source/get-started/cloud-install/azure.rst [hunk 4]
```diff
@@ -1,9 +1,32 @@
...
 
-|CL-ATTR| on Microsoft Azure
-############################
+|CL-ATTR| on Microsoft\* Azure\*
+################################
 
-|CL-ATTR| is available for you to use in the Microsoft* Azure* marketplace and
+The Azure :abbr:`CLI (Command Line Interface)` offers the ability to create and
+manage resources in MS Azure from the command line. In this tutorial you learn
+to:
```
[the remark this answers is not in the document]
Comment (DougTW) [reply source/get-started/cloud-install/azure.rst]: Done.

**Comment by mvincerx** [comment]
LGTM. Thanks, @DougTW

**Pull request merged**